### ISSUE TITLE
Please simplify your examples in termes of referenced files

### ISSUE THREAD

**Issue by the8thavatar** [swisnl/jQuery-contextMenu#561]
I like the examples as you present them.. what I need is a working sample that is as easy to pick up and use. As it is there seem to be lots of references to external files, certain css files for instance that are not in your download and so instead of being able to pick this up and use it I have to put it off for another time when I have the time to find the parts of the sample I need to keep. Ideally each sample should be separate so that the programmer can look at this in isolation rather than getting all examples and having to try ripping bits out to see what will still work.

**Comment by bbrala** [comment]
Yeah you are right, it could be beneficial to sandbox them a bit more to make things easier.
